data/socialwidgets.js

```diff
@@ -214,7 +226,11 @@ function replaceButtonWithHtmlCodeAndUnblockTracker(button, tracker, html) {
     // to prevent replacing an already removed button
     if (button.parentNode !== null) {
       let codeContainer = document.createElement("div");
-      codeContainer.innerHTML = html;
+      if(typeof html == "string") {
+        codeContainer.innerHTML = html;
```

Comment:
can you explain or maybe put a comment in explaining this? (and the following lines)

I'm not 100% what innerHTML and outerHTML is doing